clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -27,7 +27,8 @@ using namespace cir;
 
 /// Given an expression of pointer type, try to
 /// derive a more accurate bound on the alignment of the pointer.
-Address CIRGenFunction::emitPointerWithAlignment(const Expr *expr) {
+Address CIRGenFunction::emitPointerWithAlignment(const Expr *expr,
+                                                 LValueBaseInfo *baseInfo) {
```

Comment:
We have a new warning from the previous PR related to unused variable, should we quickly handle it here?

```
if (auto const *ece = dyn_cast<ExplicitCastExpr>(ce)) { <----- ece
   cgm.errorNYI(expr->getSourceRange(),
                   "emitPointerWithAlignment: explicit cast");
   return Address::invalid();
}
```

Reply:
No. It's best to keep changes like that separate to avoid mingling unrelated changes in the commit history. We wouldn't want a warning to be reintroduced if the commit for this PR were reverted, for example. Thanks for brining this to my attention though, I'll take care of it.

Reply:
Yes, that make sense